src/command_modules/azure-cli-find/azure/cli/command_modules/find/custom.py

```diff
@@ -0,0 +1,116 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from __future__ import print_function
+
+import os
+import textwrap
+import shutil
+
+import six
+from whoosh.highlight import UppercaseFormatter, ContextFragmenter
+from whoosh.qparser import MultifieldParser
+from whoosh import index
+from whoosh.fields import TEXT, Schema
+
+from azure.cli.command_modules.find._gather_commands import build_command_table
+import azure.cli.core.azlogging as azlogging
+
+logger = azlogging.get_az_logger(__name__)
+
+INDEX_PATH = os.path.join(os.path.expanduser('~'), '.azure', 'search_index')
```

Comment:
Can you use `from azure.cli.core._environment import get_config_dir`? Not everyone uses ~/.azure